tasks/deploy-s3.js

```diff
@@ -35,7 +35,7 @@ function deployS3(config) {
         }),
       )
       // gzip, Set Content-Encoding headers and add .gz extension
-      .pipe(aws.gzip())
+      .pipe(gulpIf(file => !file.path.match(/\.mp4$/), aws.gzip()))
```

Comment:
We might need to add more video file extensions if it's relevant for others.

Reply:
👍